DependencyInjection/ClientFactory.php

```diff
@@ -7,6 +7,11 @@
 use Bugsnag\Callbacks\CustomUser;
 use Bugsnag\Client;
 use Bugsnag\Configuration as Config;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
```

Comment:
Hmm, we probably don't want to force this on people. What if people are running on a system where they cannot write to the disk, not just for security, but for a serverless cloud deployment, etc.

Reply:
True.  I'm not sure how we'd proceed with storing session counts between sessions without it tho.  We could allow the user to specify the adapter, but without one we don't want to be firing off posts every time a request is made.